apps/admin/src/pages/Users/Users.tsx

```diff
@@ -0,0 +1,304 @@
+                  rel='noreferrer'
+                  className='flex flex-row gap-[10px] items-center'
+                >
+                  <span className='w-full flex flex-row items-center gap-[5px]'>
+                    <i className='w-[20px] h-[20px]'>
+                      <GithubIcon className='fill-[#a3aed0]' />
+                    </i>
+                    <span className='text-gray-700 font-bold text-[14px]'>
+                      {info?.row?.original?.github}
+                    </span>
+                  </span>
+                </a>
+              </div>
+            </div>
+          </div>
+        );
+      },
+    }),
+    columnHelper.accessor('projectsCount', {
+      header: () => 'Projects',
+      cell: (info) => info.getValue(),
+    }),
+    columnHelper.accessor('createdAt', {
+      header: () => 'Created At',
+      cell: (info) => new Date(info?.getValue()).toDateString(),
+      size: 200,
+    }),
+    columnHelper.accessor('role', {
+      header: () => 'Role',
+      size: 200,
+      cell: (info) => {
+        const selectedValue = {
+          value: info.getValue(),
+          label: info.getValue(),
+        };
+
+        return (
+          <span className='text-gray-700 font-bold text-[14px]'>
+            <Select
+              isSearchable={false}
+              options={options}
+              value={selectedValue}
+              styles={styles}
+              hideSelectedOptions
+              onChange={({ value }) => {
+                updateUser(info?.row?.original, value, 'ROLE');
+              }}
+            />
+          </span>
+        );
+      },
+    }),
+    columnHelper.display({
+      id: 'actions',
+      header: 'Actions',
+      cell: (info) => {
+        return (
+          <button
+            className={classNames(
+              'text-[14px] text-white font-bold py-[5px] px-[20px] rounded-full',
+              {
+                'bg-red-600': info?.row?.original?.banned == false,
+                'bg-green-600': info?.row?.original?.banned == true,
+              }
+            )}
+            onClick={() => {
+              updateUser(info?.row?.original, undefined, 'BAN');
+            }}
+          >
+            {info?.row?.original?.banned ? 'Unban' : 'Ban'}
+          </button>
+        );
+      },
+    }),
+  ];
+
+  const instance = useReactTable({
+    data: data?.getAllUsersAdmin?.results ?? [],
+    columns,
+    manualPagination: true,
+    getCoreRowModel: getCoreRowModel(),
+    manualSorting: true,
+    onSortingChange: setSorting,
+    state: {
+      sorting,
+    },
+  });
+
+  return (
+    <div className='w-full h-full bg-white p-[30px] flex flex-col gap-[20px]'>
+      <p className='text-gray-900 text-3xl font-bold'>Users</p>
+      <input
+        type='text'
+        placeholder='Search'
+        value={search}
+        className='w-full h-[50px] rounded-[10px] border-[1px] border-gray-300 p-[10px] focus:outline-none focus:border-[#3f51b5]'
```

Comment:
add colors to tailwind config or use a default one

Reply:
done